index.html

```diff
@@ -11,51 +11,187 @@
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="about-me">
+            <div class="section-title">
+                <h2 class="heading">About Me</h2>
+            </div>
+            <div class="information">
+                <p class="text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="btn">
+                <button type="button">Get My Resume</button>
+            </div>
```

Comment:
- [ ] It would be great to apply the button interactions, as shown above, 👍 for improving the User Interface.